### PR TITLE
Add AUR package to README.md

### PR DESCRIPTION
**Pull request by aarch-angel**
Hello! Thanks for making this, I've been using it for years and absolutely love it. I've created an [AUR package](https://aur.archlinux.org/packages/nier-cursors-bin) to allow system wide installation on Arch Linux via the package manager.

All this PR does is add a note about this to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Adwaita is a fallback for those not included. This can be changed in the "index.
 ## Installing
 Download the [release](https://github.com/Beinsezii/NieR-Cursors/releases) archive for your operating system
  - Linux: Unpack the archive and put the folder in your user icons folder (usually `~/.icons/`), then pick the cursor theme with whatever desktop/theme manager you use.
+     - Arch Linux has an [AUR package](https://aur.archlinux.org/packages/nier-cursors-bin) available to install the cursors system wide.
```
Comment (Beinsezii) [inline README.md]:
```suggestion
     - Arch Linux has a 3rd-party [AUR package](https://aur.archlinux.org/packages/nier-cursors-bin) available to install the cursors system wide.
```

**Review by Beinsezii — CHANGES_REQUESTED** [review]

**Comment by Beinsezii** [comment]
Pretty cool package. I'll have to migrate it from the ancient blender version at some point so a `-git` version is viable too.

**Comment by aarch-angel** [comment]
Yes, I was thinking about looking into that too, just don't really have the time at the moment unfortunately.

Maybe there's possiblity for user colour customisation with a git package? Not sure how much else can be added to an already pretty fleshed out set of cursors though.

Thanks again for making this! :)

**Comment by Beinsezii** [comment]
> Maybe there's possiblity for user colour customisation with a git package?

Unless they overhauled the CLI with 3X or 4X, the most you can do in blender is pick a scene to render. That's why everything is duplicated into a bunch of single object scenes.

You could *maybe* just like regex the blend file and swap out the compositor tint RGB?

Or remake the computer screen effect in pure ImageMagick which is done on CLI, then blender will just render the base grayscales.

>  Not sure how much else can be added to an already pretty fleshed out set of cursors though.

I still have to make a pen cursor but I don't even know anything on Linux I can test it on. Windows uses it for some ancient input fields I think, which is likely why it was requested.

**Comment by aarch-angel** [comment]
> I still have to make a pen cursor but I don't even know anything on Linux I can test it on. Windows uses it for some ancient input fields I think, which is likely why it was requested.

I have a Linux laptop with a touchscreen and pen, so this is actually something I'd be interested in, and would be happy to test! Is there even an appropriate cursor from the game though? If not, I could *try* making a custom one in the game's style when I end up having more time.

**Comment by Beinsezii** [comment]
> Is there even an appropriate cursor from the game though?

IIRC there's a `Writing Implement` junk item. It's been years since I played but after 200%'ing the game I think my memory is good enough.

If blender 2.8 -> 4.3 turns out to not be that bad, I could maybe do that this weekend. The main thing I worry about is I did lots of Freestyle buggery to get the outlines the way they are, with the slices still cut out. I would eventually like to just port the whole thing to geometry nodes since the shapes are mostly just simple polygons and primitives.

Update: it's the weekend, Freestyle is completely wrong now in 4.2.3 LTS. Loving life.

**Pull request merged**